### ISSUE TITLE
test_file_jpeg2k.py::TestFileJpeg2k::test_16bit_monchrome_j2k_like_tiff FAILED on ppc64

### ISSUE THREAD

**Issue by ernsteiswuerfel** [python-pillow/Pillow#3290]
On my PowerMac G5 (Gentoo Linux, ppc64, big endian) test_16bit_monchrome_j2k_like_tiff test fails with Pillow 5.2.0 and git-master.

### What did you do?
`make release-test`

### What versions of Pillow and Python are you using?
Pillow: git-master
Python: 3.6.5

```
Tests/test_file_jpeg2k.py::TestFileJpeg2k::test_16bit_j2k_roundtrips PASSED
Tests/test_file_jpeg2k.py::TestFileJpeg2k::test_16bit_jp2_roundtrips PASSED
Tests/test_file_jpeg2k.py::TestFileJpeg2k::test_16bit_monchrome_j2k_like_tiff FAILED

==================================================================================================== FAILURES =====================================================================================================
________________________________________________________________________________ TestFileJpeg2k.test_16bit_monchrome_j2k_like_tiff ________________________________________________________________________________

self = <test_file_jpeg2k.TestFileJpeg2k testMethod=test_16bit_monchrome_j2k_like_tiff>

    def test_16bit_monchrome_j2k_like_tiff(self):
    
        tiff_16bit = Image.open('Tests/images/16bit.cropped.tif')
        j2k = Image.open('Tests/images/16bit.cropped.j2k')
>       self.assert_image_similar(j2k, tiff_16bit, 1e-3)

Tests/test_file_jpeg2k.py:159: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
Tests/helper.py:154: in assert_image_similar
    raise e
Tests/helper.py:146: in assert_image_similar
    ave_diff, epsilon))
E   AssertionError: 0.001 not greater than or equal to 254.937744140625 :  average pixel value difference 254.9377 > epsilon 0.0010
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! Interrupted: stopping after 1 failures !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
================================================================================ 1 failed, 269 passed, 5 skipped in 29.63 seconds =================================================================================
make: *** [Makefile:81: release-test] Fehler 2
```

**Comment by ernsteiswuerfel** [comment]
Still fails in git-master.

```
--------------------------------------------------------------------
PIL SETUP SUMMARY
--------------------------------------------------------------------
version      Pillow 5.4.0.dev0
platform     linux 3.6.5 (default, Dec 15 2018, 17:39:35)
             [GCC 7.4.0]
--------------------------------------------------------------------
--- JPEG support available
--- OPENJPEG (JPEG2000) support available (2.3)
--- ZLIB (PNG/ZIP) support available
*** LIBIMAGEQUANT support not available
--- LIBTIFF support available
--- FREETYPE2 support available
--- LITTLECMS2 support available
--- WEBP support available
--- WEBPMUX support available
--------------------------------------------------------------------

[...]

==================================================================================================== FAILURES =====================================================================================================
_______________________________________________________________________________ TestFileJpeg2k.test_16bit_monochrome_j2k_like_tiff ________________________________________________________________________________

self = <test_file_jpeg2k.TestFileJpeg2k testMethod=test_16bit_monochrome_j2k_like_tiff>

    def test_16bit_monochrome_j2k_like_tiff(self):
    
        tiff_16bit = Image.open('Tests/images/16bit.cropped.tif')
        j2k = Image.open('Tests/images/16bit.cropped.j2k')
>       self.assert_image_similar(j2k, tiff_16bit, 1e-3)

Tests/test_file_jpeg2k.py:175: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
Tests/helper.py:154: in assert_image_similar
    raise e
Tests/helper.py:146: in assert_image_similar
    ave_diff, epsilon))
E   AssertionError: 0.001 not greater than or equal to 254.937744140625 :  average pixel value difference 254.9377 > epsilon 0.0010
================================================================================ 1 failed, 283 passed, 5 skipped in 15.01 seconds =================================================================================
make: *** [Makefile:81: release-test] Fehler 1

```

**Comment by mattst88** [comment]
Probably big-endian related. Fails on hppa as well.

**Comment by ernsteiswuerfel** [comment]
Still fails in 5.4.1.

**Comment by radarhere** [comment]
Closing, as this is now part of #4213

**Comment by radarhere** [comment]
This has now been fixed by #5901
